client-tools/Shell/V8ClientConnection.cpp

```diff
@@ -987,6 +987,99 @@ static void ClientConnection_httpPostRaw(
   ClientConnection_httpPostAny(args, true);
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// @brief ClientConnection method wrap single fuzz request
+////////////////////////////////////////////////////////////////////////////////
+
+static void httpFuzzRequest(V8ClientConnection* v8connection,
```

Comment:
Please only compile this conditionally, if `ARANGODB_ENABLE_FAILURE_TESTS` is set.

Reply:
I think this function is only called from one place in the code, and almost does nothing, so it can be removed and its relevant line
```
v8connection->requestFuzz(isolate, fuzzer)
```
can be put at the call site.

Reply:
Addressed with ifdef